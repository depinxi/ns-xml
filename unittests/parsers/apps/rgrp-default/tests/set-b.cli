-c 'Set the exclusive group argument'
