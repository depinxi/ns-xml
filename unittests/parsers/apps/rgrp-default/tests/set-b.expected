CLI: "-c", "Set the exclusive group argument"
Value count: 0
Values: 
Error count: 0
Subcommand: 
mainGroup=
exclusiveSubGroup=
xGroupSwitchA=False
xGroupArgumentB=
requiredGroup=
rGroupMultiArgC=Set the exclusive group argument
rGroupSwitchD=False
exclusiveRequiredGroup=xrGroupSwitchE
xrGroupSwitchE=True
xrArgumentF=
